pyodide_build/pywasmcross.py

```diff
@@ -256,6 +260,34 @@ def handle_command(line, args, dryrun=False):
             arg = arg[:-3] + '.wasm'
             output = arg
 
+        # Fix for scipy to link to the correct BLAS/LAPACK files
+        if arg.startswith('-L') and 'CLAPACK-WA' in arg:
```

Comment:
The overall goal of this is to convert,
```
-L../../../../CLAPACK/CLAPACK-WA
```
into
```
../../../../CLAPACK/CLAPACK-WA/blas_WA.bc
```
and (optionally with a second link to LAPACK when it's needed).